config/rabbitmq.php

```diff
@@ -24,6 +24,46 @@
      */
     'factory_class' => Enqueue\AmqpLib\AmqpConnectionFactory::class,
 
+    /*
```

Comment:
Lets put this whole block as a last key of config

Reply:
Ok. I will move this to the end